### DISCUSSION TITLE
How to update image and update container?

### DISCUSSION THREAD

**Discussion by Digital-Media-Group** [Dokploy/dokploy#764] [General]
I don't see in the documentation any specification on how to update the image or the new version of a template.

How could I update? Is there any option?

**Comment by Siumauricio** [comment]
Well you would have to go to the documentation of the template you want to update, and simply update the image tag, this you can do it yourself at the end of the day are docker-compose files so you can modify them to your liking.

**Reply by Digital-Media-Group** [reply]
Thank you for your reply. I understand, but if the image is the last :latest when shutting down and unfolding again it does not update, nor when rebuilding. How should it be done from the panel? ...

**Comment by Digital-Media-Group** [comment]
Someone can help me with that doubt. Why doesn't the last image download when turning off and then on?
